### ISSUE TITLE
Empty cell is not announced as blank.

### ISSUE THREAD

**Issue by Bilal3120** [SAP/ui5-webcomponents-react#3133]
### **Bug Description**
Empty cells are not announced as blank in analytical table.

### **Expected Behavior**
The empty cells should be announced as empty or blank.

### **Steps to Reproduce**
1. Go to [CodeSandbox](https://codesandbox.io/s/recursing-cray-kq32lr?file=/src/App.js)
2. Focus on empty cell in the table.

### **Context**
- UI5 Web Components version:1.4.0
- OS/Platform: Mac OS
- Browser: Chrome
- Affected component: Analytical table

### **Log Output / Stack Trace / Screenshots**
![MicrosoftTeams-image (5)](https://user-images.githubusercontent.com/93638146/175878902-7d65c659-4fb3-42ff-9172-f6ccb5acd6a8.png)

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

**Comment by MarcusNotheis** [comment]
Hey @Bilal3120,
do you have a link where this accessibility standard is described that empty cells should be announced?
I've just tried to look at the Announcements of the [UI5 Grid Table](https://ui5.sap.com/#/entity/sap.ui.table.Table/sample/sap.ui.table.sample.Resizing) and it looks like there are no announcements of empty cells as well:
<img width="2020" alt="Screenshot 2022-06-29 at 14 33 46" src="https://user-images.githubusercontent.com/8775211/176437560-5c2e6cca-4de4-4cce-a6cb-382375d29e28.png">

**Comment by Bilal3120** [comment]
Hi @MarcusNotheis,
I have checked with the Accessibility team on this issue. Since it is a limitation from UI5 they said that we can close this issue now. It seems that they are tracking the same issue with UI5 team in another ticket. So closing this issue as per their comments on the requirement.

Thanks and regards,
Bilal Ahmed Pasha
